README.md

```diff
@@ -1,18 +1,32 @@
 # React Chat Example
 
-![Test](https://github.com/xmtp/example-chat-react/actions/workflows/test.yml/badge.svg)
-![Lint](https://github.com/xmtp/example-chat-react/actions/workflows/lint.yml/badge.svg)
-![Build](https://github.com/xmtp/example-chat-react/actions/workflows/build.yml/badge.svg)
+![Status](https://img.shields.io/badge/Project_Status-Archived-red)
 
 ![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
 
+The XMTP React Chat example app has been archived and this repo is no longer being maintained.
+
+You can still fork this repo and existing forks continue to work.
+
+Check out these other supported app repos you can use for guidance and inspiration when building with XMTP:
+
+- The [XMTP Quickstart React chat app](https://github.com/xmtp/xmtp-quickstart-react) demonstrates the core capabilities of the XMTP client SDK, providing code you can use to learn to build a basic messaging app.
+- The [XMTP Inbox chat app](https://github.com/xmtp-labs/xmtp-inbox-web) demonstrates core and advanced capabilities of the XMTP client SDK, showcasing innovative ways of building with the XMTP client SDK. This app is an evolution of the XMTP React Chat example app previously maintained in this repo.
```

Comment:
What do you think of the positioning of the differences between the quickstart and inbox apps? Please feel free to suggest updates!

Reply:
I think as a small non-blocking note I would rather point people to the new repos before mentioning you can still fork this one just because those will be actively maintained. But like I said, not blocking!

Reply:
OMG @elisealix22 - SO GOOD. Thank you - updating now....